boards/makerbase/index.rst

```diff
@@ -0,0 +1,10 @@
+.. _boards-makerbase:
+
+Makerbase
```

Comment:
```suggestion
MakerBase
```

Reply:
They call them `Makerbase`, please take a look here: https://makerbase.com.cn/en/products/

Reply:
Apparently they cannot make up their mind regarding this - I looked them up on their online store, they use "MakerBase" there: https://makerbase3d.com

Let's stick with "Makerbase".